nautobot/core/ui/object_detail.py

```diff
@@ -789,7 +793,14 @@ def get_extra_context(self, context: Context):
             if self.table_attribute:
                 body_content_table_queryset = getattr(instance, self.table_attribute)
             else:
-                body_content_table_queryset = body_content_table_model.objects.filter(**{self.table_filter: instance})
+                if isinstance(self.table_filter, str):
+                    table_filters = [self.table_filter]
+                else:
+                    table_filters = self.table_filter
```

Comment:
Might be more robust this way.
```suggestion
                elif isinstance(self.table_filter, list):
                    table_filters = self.table_filter
                else:
                    table_filters = []
```
```suggestion
                elif isinstance(self.table_filter, list):
                    table_filters = self.table_filter
                else:
                    table_filters = []
```